services/CollectionService/src/index.php

```diff
@@ -25,40 +22,25 @@
   'twig.path' => __DIR__.'/../templates',
 ));
 
-$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
-  return $twig;
-}));
 $islandoraResourceServiceProvider = new \Islandora\ResourceService\Provider\ResourceServiceProvider;
 
-$app->register($islandoraResourceServiceProvider);
+//Registers Resource Service and defines current app's path for config context
+$app->register($islandoraResourceServiceProvider, array(
+  'islandora.BasePath' => __DIR__,
+));
 $app->mount("/islandora", $islandoraResourceServiceProvider);
 $app->register(new \Islandora\ResourceService\Provider\UuidServiceProvider(), array(
   'UuidServiceProvider.default_namespace' => $app['config']['islandora']['defaultNamespaceDomainUuuidV5'],
 ));
 
-//$app['uuid'] = $app['islandora.uuid5'](rand());
+//We will use a static uuid5 to fake a real IRI to replace <> during turtle parsing
+$app['uuid5'] = $app->share(function () use ($app) {
```

Comment:
Might as well just jam the fake uri here.  Or even better, just store it as a variable.  No need for it to be in the DI container.  When we refactor into a thin controller, the variable should reside within the fat service the controller utilizes.